src/dynamics/video_player/player/player.js

```diff
@@ -1257,7 +1257,10 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                  */
                 initAdsRenderFailTimeout: function() {
                     const renderTimeout = Number(this.get("adsrendertimeout"));
-                    const repeatMicroseconds = 200;
+                    let repeatMicroseconds = 200;
+                    if (renderTimeout < repeatMicroseconds) {
+                        repeatMicroseconds = renderTimeout;
```

Comment:
Check to see if renderTimeout !== undefined too

Reply:
Seems like this moving solve the issue https://github.com/betajs/betajs-media-components/blob/478af177345ce0008c660c9d52c0c82c9e232b94/src/dynamics/video_player/player/player.js#L1260-L1265